Update ExcessiveHeaders.md (#4112)
umbraco/UmbracoDocs: Extending/Health-Check/Guides/ExcessiveHeaders.md

For IIS you will need to manipulate `web.config` (If you don't have `web.config` already in your project you will need to add it at the root). Ensure to remove the custom `X-Powered-By` and `Server` header as shown in the following example.

> Note: removeServerHeader attribute was added in IIS 10.0 and does not work in versions of Windows prior to Windows Server, version 1709 or Windows 10, version 1709.

Reviewer: ```suggestion
:::note
The `removeServerHeader` attribute is added in IIS 10.0 and does not work in versions of Windows prior to Windows Server version 1709 or Windows 10 version 1709.
:::
```
